app/views/shared/_status.html.erb

```diff
@@ -1,9 +1,6 @@
 <p>
-  <%= form.label "visibility:" %><br>
+  <%= form.label "Visibility:" %><br>
   <%= form.select :status,
-                  [["Public", "public"], ["Private", "private"], ["Archived", "archived"]],
-                  selected: "public" %>
-  <%= element.errors.full_messages_for(:status).each do |message| %>
-  <div><%= message %></div>
-<% end %>
+                  [['Public', 'public'], ['Personal', 'personal'], ['Archived', 'archived']],
```

Comment:
@JsMontesR What if we move this trivial logic into helper methods? Does it make sense?

Reply:
That make sense for me, maybe with the helper strategy we can access the concern statuses and render them here. That would be better I think. 🤔

Reply:
@JsMontesR I think it would be a better approach!

Reply:
✅ Done!